packages/discord.js/src/util/Components.js

```diff
@@ -42,29 +42,3 @@
 /**
  * @typedef {ActionRowData|ButtonComponentData|SelectMenuComponentData} ComponentData
  */
-
-class Components extends null {
```

Comment:
a file with only typedefs?

Reply:
Yeah

Reply:
You can move them to Util, this feels weird like that